client/blocks/term-form-dialog/index.jsx

```diff
@@ -264,28 +266,42 @@ class TermFormDialog extends Component {
 
 		return (
 			<FormFieldset>
-				<FormLegend>{ labels.parent_item }</FormLegend>
 				<FormLabel>
-					<FormCheckbox
-						ref="topLevel"
-						checked={ this.state.isTopLevel }
-						onChange={ this.onTopLevelChange }
-					/>
+					<FormToggle checked={ isTopLevel } onChange={ this.onTopLevelChange } />
 					<span>
-						{ translate( 'Top level', { context: 'Terms: New term being created is top level' } ) }
+						{ translate( 'Top level %(term)s', {
+							args: { term: labels.singular_name },
+							context: 'Terms: New term being created is top level',
+						} ) }
 					</span>
+					{ isTopLevel && (
+						<span className="term-form-dialog__top-level-description">
+							{ translate( 'Disable to select a %(parentTerm)s', {
+								args: { parentTerm: labels.parent_item },
```

Comment:
Could you add a `comment` with examples of what the variable `parentTerm` could be? i.e. examples for "term". Please add it also to the other translatable strings where applicable. Thanks!

Reply:
Added comments in c1203ec